src/seedu/addressbook/commands/FindCommand.java

```diff
@@ -47,10 +47,16 @@ public CommandResult execute() {
      * @return list of persons found
      */
     private List<ReadOnlyPerson> getPersonsWithNameContainingAnyKeyword(Set<String> keywords) {
+        final ArrayList<String> keywordsLowerCase = new ArrayList<String>(keywords);
+        keywordsLowerCase.replaceAll(String::toLowerCase);
```

Comment:
Nice hack you used to transform the inputs in the list to lowercase while preserving the order of elements in the list.